File: 3285_find_indices_of_stable_mountains.py
class Solution:
    def stableMountains(self, height: list[int], threshold: int) -> list[int]:
        ret_list = []
        for i in range(1, len(height)):
            if height[i - 1] > threshold:
                ret_list.append(i)
        return ret_list

File: JavaScript/3285_find_indices_of_stable_mountains.js
/**
 * @param {number[]} height
 * @param {number} threshold
 * @return {number[]}
 */
var stableMountains = function(height, threshold) {
    let answer = [];
    for (let i = 1; i < height.length; i++) {
        if (height[i - 1] > threshold) {
            answer.push(i);
        }
    }
    return answer;
};

